node/cn/handler.go

```diff
@@ -367,6 +370,11 @@ func (pm *ProtocolManager) Stop() {
 	logger.Info("Klaytn protocol stopped")
 }
 
+// SetSyncStop sets value of syncStop flag. If it's true, peer sync process does not proceed.
+func (pm *ProtocolManager) SetSyncStop(flag bool) {
+	pm.syncStop = flag
```

Comment:
It seems that writing this variable and reading it can be done by different thread. In this case we need to use an atomic operation.

Reply:
changed SetSyncStop method atomically.